gapis/api/cmd_id_group.go

```diff
@@ -60,6 +61,14 @@ func NewRoot(idx []uint64) *SubCmdRoot {
 		SubGroup: CmdIDGroup{Name: "Subgroup"}}
 }
 
+// SubCmdMarkerGroup contains the range info of a marker group of subcommands.
+type SubCmdMarkerGroup struct {
```

Comment:
Or maybe we can just use `CmdIDGroup` directly?

Reply:
F2F discussion: Will be removed.

Reply:
Fixed in Fixed in #1064